models/general/knowledge/en/currency_to_bitcoin.txt

```diff
@@ -53,3 +53,118 @@ convert bitcoin to jpy| convert btc to jpy| convert btc to japanese yen| convert
+!console:Latest exchange rate is 1 BTC = $last$ NZD
+{
+"url":"https://blockchain.info/ticker",
+"path":"$.NZD"
+}
+eol
+
+convert bitcoin to brazilian real | convert btc to brl| convert btc to brazilian real | convert btc to brazilian real | btc to brazilian real |btc to brazilian real |btc to brazilian real |what is latest exchange rate for brazilian real  to bitcoin|what is latest exchange rate for brazilian real to BTC | can you convert bitcoin to brazilian real | can you convert btc to brazilian real | can you convert btc to brl| can you convert btc to brl| btc to brl|can you tell me what is latest exchange rate for brl to bitcoin|can you tell me what is latest exchange rate for brl to btc |could you tell me what is latest exchange rate for brl to btc |could you convert bitcoin to brl| could you convert btc to brl|  could you convert btc to brazilian real | btc to brazilian real |could you tell me what is latest exchange rate for brazilian real to bitcoin|could you tell me what is latest exchange rate for brl to btc | do you know what is latest exchange rate for brl to btc | can you convert bitcoin to brl| can you convert btc to brl| can you convert btc to brazilian real | btc to brazilian real |btc to brl|do you know the latest exchange rate for brazilian real  to bitcoin|do you know what is the latest exchange rate for brazilian real  to btc
+!console:Latest exchange rate is 1 BTC = $last$ BRL
+{
+"url":"https://blockchain.info/ticker",
+"path":"$.BRL"
+}
+eol
+
+convert bitcoin to russian ruble | convert btc to rub| convert btc to russian ruble | convert btc to russian ruble | btc to russian ruble |btc to russian ruble |btc to russian ruble |what is latest exchange rate for russian ruble  to bitcoin|what is latest exchange rate for russian ruble to BTC | can you convert bitcoin to russian ruble | can you convert btc to russian ruble | can you convert btc to rub| can you convert btc to rub| btc to rub|can you tell me what is latest exchange rate for rub to bitcoin|can you tell me what is latest exchange rate for rub to btc |could you tell me what is latest exchange rate for rub to btc |could you convert bitcoin to rub| could you convert btc to rub|  could you convert btc to russian ruble | btc to russian ruble |could you tell me what is latest exchange rate for russian ruble to bitcoin|could you tell me what is latest exchange rate for rub to btc | do you know what is latest exchange rate for rub to btc | can you convert bitcoin to rub| can you convert btc to rub| can you convert btc to russian ruble | btc to russian ruble |btc to rub|do you know the latest exchange rate for russian ruble  to bitcoin|do you know what is the latest exchange rate for russian ruble  to btc
```

Comment:
There is some redundancy in questions kindly fix that 
like btc to russian ruble |btc to russian ruble